lib/cylc/prompt.py

```diff
@@ -17,14 +17,32 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import sys
+import gtk
```

Comment:
This line adds `gtk` to `cylc poll`, which is causing `tests/cylc-poll/03-poll-all.t` to fail in my environment.

I am getting a traceback like this in the `poll_now.20141207T0000Z` task of the test.

```
Traceback (most recent call last):
  File "/home/matt/cylc.git/bin/cylc-poll", line 26, in <module>
    from cylc.prompt import prompt
  File "/home/matt/cylc.git/lib/cylc/prompt.py", line 20, in <module>
    import gtk
  File "/usr/lib64/python2.6/site-packages/gtk-2.0/gtk/__init__.py", line 64, in <module>
    _init()
  File "/usr/lib64/python2.6/site-packages/gtk-2.0/gtk/__init__.py", line 52, in _init
    _gtk.init_check()
RuntimeError: could not open display
```